libs/user-activity/interface/index.ts

```diff
@@ -0,0 +1,9 @@
+export interface User_Activity {
```

Comment:
Let's follow consistency in Interface naming convention.

Reply:
Resolved.